packages/gestalt/src/Controller.js

```diff
@@ -87,32 +114,14 @@ export default class Controller extends React.Component<Props, State> {
     this.updateTriggerRect(this.props);
```

Comment:
question here - could we actually just do a `this.forceUpdate()` here since that would ultimately result in `this.updateTriggerRect` being called from gDSFP?

Reply:
`forceUpdate` seems a little smelly to me but I don't think I can defend the position. Seems the same in this case. Do you feel like that's a cleaner solution?